quickstarts/microsoft.apimanagement/api-management-create-with-external-vnet/README.md

```diff
@@ -13,6 +13,8 @@
 [![Deploy To Azure US Gov](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazuregov.svg?sanitize=true)](https://portal.azure.us/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fquickstarts%2Fmicrosoft.apimanagement%2Fapi-management-create-with-external-vnet%2Fazuredeploy.json)
 [![Visualize](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/visualizebutton.svg?sanitize=true)](http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fquickstarts%2Fmicrosoft.apimanagement%2Fapi-management-create-with-external-vnet%2Fazuredeploy.json)
 
+**IMPORTANT - This quickstart template is deprecated and is replaced by [Deploy API Management in external virtual network with public IP](../api-management-create-with-external-vnet-publicip).**
```

Comment:
can we just remove it then?

Reply:
The engineering team recommends keeping if for a short time (a few months max) because some customers who are running on a legacy platform may still need to use it.